server/webhook.go

```diff
@@ -217,6 +223,16 @@ func (s *Server) registerWebhookRoutes(g *echo.Group) {
 	})
 }
 
+// validateGitHubWebhookSignature256 returns true if the signature matches the
+// HMAC hex digested SHA256 hash of the body using the given key.
+func validateGitHubWebhookSignature256(signature, key string, body []byte) bool {
+	signature = strings.TrimPrefix(signature, "sha256=")
+	m := hmac.New(sha256.New, []byte(key))
+	m.Write(body)
+	got := hex.EncodeToString(m.Sum(nil))
+	return subtle.ConstantTimeCompare([]byte(signature), []byte(got)) == 1
```

Comment:
https://docs.github.com/en/developers/webhooks-and-events/webhooks/securing-your-webhooks#validating-payloads-from-github

<img width="831" alt="CleanShot 2022-07-24 at 21 52 02@2x" src="https://user-images.githubusercontent.com/2946214/180650235-385799d2-cdf4-4c68-a542-0a1cb1bad052.png">